### ISSUE TITLE
The first, fourth and fifth quiz modes don't work

### ISSUE THREAD

**Issue by Naoto-Seri** [jehutyno/yomikata#7]
Hi,
The second, third and fourth quiz modes can be clicked and highlighted in orange. The first (hiragana to kanji), fifith (hiragana to writing)  and sixth (AUTO) can't be clicked, nothing happens.

![Screenshot_20230303-104912 (2)](https://user-images.githubusercontent.com/76906331/222688974-3a648207-ac4b-4087-bc16-af6a890d8e05.jpg)

**Comment by LucasDooms** [comment]
Hello, I believe this was fixed in Pull Request #3 , however the version of the app on the play store has not been updated yet.
For now, if you change these options in a different quiz category (e.g. Hiragana), then the changes will carry over to your selections.

**Comment by jehutyno** [comment]
@LucasDooms I think you're right, a new version should be released as other things might be fixed.
Would you contact me on Discord ?

**Comment by Naoto-Seri** [comment]
> Hello, I believe this was fixed in Pull Request #3 , however the version of the app on the play store has not been updated yet. For now, if you change these options in a different quiz category (e.g. Hiragana), then the changes will carry over to your selections.

Indeed, it seems the problem only appears for "Mes sélections" and not the other categories. Thank you!
